rules.txt

```diff
@@ -847,6 +847,7 @@
 /records/survey/(.*)                      301  /departments/department-of-records/
 /recovery                                 301  /programs/covid-19-recovery-office/
 /recoverPHL                               301  /programs/covid-19-recovery-office/
+/recovery/index.html                      301  /departments/covid-19-recovery-office/
```

Comment:
let change this from /recovery/index.html to /recovery/(.*) so that the index.html and any other pages on that site get redirected to the covider 19 office plz